modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/dcrm/api/util/DCRUtils.java

```diff
@@ -0,0 +1,102 @@
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.identity.integration.test.oauth2.dcrm.api.util;
+
+import org.apache.commons.codec.binary.Base64;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+import org.json.simple.parser.JSONParser;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Utils for preparing DCR payloads
+ */
+public class DCRUtils {
+
+    private static JSONParser parser = new JSONParser();
+    private static final String REGISTER_REQUESTS_LOCATION = "registration.requests.location";
+
+    /**
+     * Get register request JSON object.
+     *
+     * @param fileName File name.
+     * @return Register request JSON object.
+     * @throws Exception Exception.
+     */
+    public static JSONObject getRegisterRequestJSON(String fileName) throws Exception {
+
+        return (JSONObject) parser.parse(new FileReader(getFilePath(REGISTER_REQUESTS_LOCATION, fileName)));
+    }
+
+    /**
+     * Get file path.
+     *
+     * @param folderPath Folder path.
+     * @param fileName   File name.
+     * @return File path.
+     * @throws Exception Exception.
+     */
+    public static String getFilePath(String folderPath, String fileName) throws Exception {
+
+        Path path = Paths.get(System.getProperty(folderPath) + fileName);
+        if (!Files.exists(path)) {
+            throw new FileNotFoundException("Failed to find file: " + path.toString());
+        }
+        return path.toString();
+    }
+
+    public static void setRequestHeaders(HttpPost request, String username, String password) {
+
+        request.addHeader(HttpHeaders.AUTHORIZATION, getAuthzHeader(username, password));
+        request.addHeader(HttpHeaders.CONTENT_TYPE, OAuthDCRMConstants.CONTENT_TYPE);
+    }
+
+    public static String getPath(String tenant) {
+
+        if (tenant.equals("carbon.super")) {
+            return OAuthDCRMConstants.DCR_ENDPOINT_HOST_PART + OAuthDCRMConstants.DCR_ENDPOINT_PATH_PART;
+        } else {
+            return OAuthDCRMConstants.DCR_ENDPOINT_HOST_PART + "/t/" + tenant + OAuthDCRMConstants
+                    .DCR_ENDPOINT_PATH_PART;
+        }
+    }
+    public static String getAuthzHeader(String username, String password) {
+
+        return "Basic " + Base64.encodeBase64String((username + ":" + password).getBytes()).trim();
```

Comment:
Let's use a constant
ref: https://github.com/wso2/product-is/blob/dfab6bbaaf4db4cbce513eed4f45f815dd11c251/modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/utils/OAuth2Constant.java#L92

Reply:
changed